docs/user_guide.md

```diff
@@ -252,6 +256,95 @@ The images can be filtered by labels:
+
+## Set Project Quotas
+
+To exercise control over resource use, as a system administrator you can set  quotas on projects. You can limit the number of artifacts that a project can contain and limit the amount of storage capacity that a project can consume. You can set default quotas that apply to all projects globally. You can also set quotas on individual projects. If you set global default quotas and also set different quotas on individual projects, the per-project quotas override the global default quotas.
+
+By default, all projects have unlimited quotas for both artifacts and storage use. 
+
+1. Go to **Configuration** > **Project Quotas**.
+   ![Project quotas](img/project-quota1.png)
+1. To set global default quotas on all projects, click **Edit**.
+   ![Project quotas](img/project-quota2.png)
+   1. For **Default artifact count**, enter the maximum number of artifacts that any project can contain. 
+   
+   Enter `-1` to set the default to unlimited. 
+   1. For **Default storage consumption**, enter the maximum quantity of storage that any project can consume, selecting `MB`, `GB`, or `TB` from the drop-down menu. 
+   
+   Enter `-1` to set the default to unlimited.
+   ![Project quotas](img/project-quota3.png)
+   1. Click **OK**.
+1. To set quotas on an individual project, click the 3 vertical dots next to a project and select **Edit**.
+   ![Project quotas](img/project-quota4.png)
+   1. For **Default artifact count**, enter the maximum number of artifacts that this individual project can contain, or enter `-1` to set the default to unlimited. 
+   1. For **Default storage consumption**, enter the maximum quantity of storage that this individual project can consume, selecting `MB`, `GB`, or `TB` from the drop-down menu.
+
+After you set quotas, the you can see how much of their quotas each project has consumed in the **Project Quotas** tab.
+
+![Project quotas](img/project-quota5.png)
+
+### How Harbor Calculates Resource Usage
+
+When setting project quotas, it is useful to know how Harbor calculates artifact numbers and storage use, especially in relation to image pushing, retagging, and garbage collection.
+
+- Harbor computes image size when blobs and manifests are pushed from the Docker client.
+- Harbor computes artifact counts when manifests are pushed from the Docker client.
+- Shared blobs are only computed once per project. In Docker, blob sharing is defined globally. In Harbor, blob sharing is defined at the project level. As a consequence, overall storage usage can be greater than the actual disk capacity.
+- Retagging images reserves and releases resources: 
+  -  If you retag an image within a project, the artifact count increases by one, but storage usage does not change because there are no new blobs or manifests.
+  - If you retag an image from one project to another, the artifact count and storage usage both increase.

```

Comment:
So we can safely say that artifact count = tag count, correct? @wy65701436

Reply:
If artifact == tag, we should state that somewhere. If things other than tags count as artifacts, we should state that and list them. @wy65701436 can you please clarify?

Reply:
@stuclem It is the same thing, I verified with @wy65701436 today

Reply:
Replaced "artifact" with "tag" as appropriate.